Node/Generator/test/app.js

```diff
@@ -1,22 +0,0 @@
-'use strict';
```

Comment:
file removed - the tests are now located in 'bot_js.test.js' and 'bot_ts.test.js'